etna/transforms/feature_selection/base.py

```diff
@@ -42,4 +45,21 @@ def transform(self, df: pd.DataFrame) -> pd.DataFrame:
         rest_columns = set(df.columns.get_level_values("feature")) - set(self._get_features_to_use(df))
         selected_columns = sorted(self.selected_features + list(rest_columns))
         result = result.loc[:, pd.IndexSlice[:, selected_columns]]
+        if self.return_features:
+            self._df_removed = df.drop(result.columns, axis=1)
         return result
+
+    def inverse_transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Apply inverse transform method of transforms to the data.
```

Comment:
I guess you take it from the TSDataset) 
It will sound more sensible if you remove "method of transforms"
Do it in filter transform too